### ISSUE TITLE
Minecraft Version: Issue Title

### ISSUE THREAD

**Issue by waseric** [kangarko/ChatControl-Pro#2352]
Minecraft Version: 1.17.1
MySQL: false
BungeeCord: true

BungeeControl 2.4.4 is installed on BungeeCord.  Works fine with CCP 8.10.11 and under.
There are two versions of BungeeControll 2.4.4 distributed (one with 8.11.0 and one with 8.10.9)

1. Drop any 8.11.0+ ChatControlPro jar into paper directory.
2. Start server.  Observe no errors during startup.
3. Enter a chat message into server's local channel.
4. Other users on same server see chat.
5. Other users spying channel from other servers do not see message.
6. Exception thrown on console, text of error log captured below.
7. Behavior repeats for any chat message.
8. Stop server
9. Drop 8.10.11 into paper directory (remove 8.11.0+).  Chat works as expected.

```error.log
------------------------------------[ 14.12.2021 21:52:00 ]-----------------------------------
ChatControl 8.11.2 encountered a FoException
Running Paper 1.17.1-R0.1-SNAPSHOT and Java 16.0.1
Plugins: HubKick v1.7.1, PlaceholderAPI v2.10.10, WorldBorder v2.1.4, EventLogger v1.4, CheckNameHistory v3.1.0, ViaVersion v4.1.1, floodgate v2.1.0-SNAPSHOT, LuckPerms v5.3.86, ProtocolLib v4.7.0, Vault v1.7.3-b131, boosCooldowns v3.17.1, Essentials v2.19.1, FastAsyncWorldEdit v1.17-410;05c22a6, SandlotHardcore v1.1, AdvancedAchievements v8.0.0, AncientGates v2.5.1, ServerSigns v4.7.0, EssentialsSpawn v2.19.1, CoreProtect v20.1, Bomberman v0.7.1, WorldGuard v7.0.6+1cff55f, GriefPrevention v16.17.1-2b69191, Skript v2.6, OreAnnouncer v2.8.3, PremiumVanish v2.7.8, DiscordSRV v1.24.0, BungeeTabListPlus v3.4.4, ChatControl v8.11.2, Winter v3.0.3, BanManager v7.5.0, Plan v5.4 build 1516
----------------------------------------------------------------------------------------------

More Information:
Cannot call tellBungee() without channel name because class org.mineacademy.chatcontrol.ChatControl does not implement getBungeeCord()!
FoException Report: Cannot call tellBungee() without channel name because class org.mineacademy.chatcontrol.ChatControl does not implement getBungeeCord()!
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.lib.fo.Valid.checkNotNull(Valid.java:60)
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.lib.fo.BungeeUtil.tellBungee(BungeeUtil.java:41)
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.model.FormattedMessage.sendBungeeChannel(FormattedMessage.java:122)
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.model.SimpleChannel.sendMessage(SimpleChannel.java:487)
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.model.SimpleChannel.sendMessage(SimpleChannel.java:335)
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.listener.ListenerChannels.execute(ListenerChannels.java:70)
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.listener.ListenerChannels.execute(ListenerChannels.java:20)
         at ChatControl-8.11.2.jar//org.mineacademy.chatcontrol.lib.fo.event.SimpleListener.execute(SimpleListener.java:89)
         at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76)
         at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
         at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628)
         at io.papermc.paper.adventure.ChatProcessor.post(ChatProcessor.java:230)
         at io.papermc.paper.adventure.ChatProcessor.processingLegacyFirst(ChatProcessor.java:115)
         at io.papermc.paper.adventure.ChatProcessor.process(ChatProcessor.java:64)
----------------------------------------------------------------------------------------------


```

Directory contents not yet uploaded -- will provide if deemed necessary.
NashornPlus was disabled for the 8.11.2 test for which this error.log was captured (based on guidance that it is no longer needed).  However, behavior is the same with or without it (and it is enabled while using 8.10.11).

**Comment by kangarko** [comment]
Remove NashornPlus and only use the latest version of ChatControl Red with the latest BungeeControl Red. Comment below if you get an error again and repost it from the latest version please.

**Comment by kangarko** [comment]
I am sorry, I noticed you were using Pro not Red. You can get your free upgrade at mineacademy.org/chatcontrol-red#upgrade

Going to have a look right now

**Comment by kangarko** [comment]
Thanks, will fix tonight.

**Comment by waseric** [comment]
> I am sorry, I noticed you were using Pro not Red. You can get your free upgrade at mineacademy.org/chatcontrol-red#upgrade
Upgrade will be complex.  It's on our to-do list for the upcoming holiday break :)

> Thanks, will fix tonight.
Thank you for the fast response.  I really like ChatControl, and your ongoing (and speedy) support is just very much appreciated!

**Comment by waseric** [comment]
confirmed fixed in 8.11.3
